src/pages/settings/Payments/PaymentsPage/BasePaymentsPage.js

```diff
@@ -257,250 +211,305 @@ class BasePaymentsPage extends React.Component {
+                        isAddPaymentMenuActive={shouldShowAddPaymentMenu}
+                        actionPaymentMethodType={shouldShowDefaultDeleteMenu || showPassword.shouldShowPasswordPrompt ? paymentMethod.selectedPaymentMethodType : ''}
+                        activePaymentMethodID={shouldShowDefaultDeleteMenu || showPassword.shouldShowPasswordPrompt ? getSelectedPaymentMethodID() : ''}
+                        listHeaderComponent={listHeaderComponent}
+                    />
+                </OfflineWithFeedback>
+            </View>
+            <AddPaymentMethodMenu
+                isVisible={shouldShowAddPaymentMenu}
+                onClose={hideAddPaymentMenu}
+                anchorPosition={{
+                    horizontal: anchorPosition.anchorPositionHorizontal,
+                    vertical: anchorPosition.anchorPositionVertical - 10,
+                }}
+                onItemSelected={(method) => addPaymentMethodTypePressed(method)}
+            />
+            <Popover
+                isVisible={shouldShowDefaultDeleteMenu}
+                onClose={hideDefaultDeleteMenu}
+                anchorPosition={{
+                    top: anchorPosition.anchorPositionTop,
+                    right: anchorPosition.anchorPositionRight,
+                }}
+            >
+                {!showConfirmDeleteContent ? (
+                    <View style={[styles.m5, !isSmallScreenWidth ? styles.sidebarPopover : '']}>
+                        {isPopoverBottomMount && (
+                            <MenuItem
+                                title={paymentMethod.formattedSelectedPaymentMethod.title || ''}
+                                icon={paymentMethod.formattedSelectedPaymentMethod.icon}
+                                description={paymentMethod.formattedSelectedPaymentMethod.description}
+                                wrapperStyle={[styles.pv0, styles.ph0, styles.mb4]}
+                                interactive={false}
+                            />
+                        )}
+                        {shouldShowMakeDefaultButton && (
                             <Button
                                 onPress={() => {
-                                    this.setState({
-                                        showConfirmDeleteContent: true,
+                                    setShouldShowDefaultDeleteMenu(false);
+
+                                    // Wait for the previous modal to close, before opening a new one. A modal will be considered completely closed when closing animation is finished.
+                                    // InteractionManager fires after the currently running animation is completed.
+                                    // https://github.com/Expensify/App/issues/7768#issuecomment-1044879541
+                                    InteractionManager.runAfterInteractions(() => {
+                                        if (Permissions.canUsePasswordlessLogins(props.betas)) {
```

Comment:
`Permissions.canUsePasswordlessLogins` was removed in https://github.com/Expensify/App/pull/22461